test/integration/org_dogs_test.rb

```diff
@@ -106,13 +106,13 @@ class OrgDogsTest < ActionDispatch::IntegrationTest
     assert_response :redirect
     follow_redirect!
     assert_equal 'Dog updated successfully.', flash[:notice]
-    assert_select "h1", "#{Dog.find(@dog_id).name}"
+    assert_select "h1", "TestDog"
```

Comment:
Where is this name coming from? I don't see it in the fixtures as a dog name? 
Do you think it would be less brittle to assert `fixture.name` instead of a string? If the name changes in the fixture the test won't break?